scribble-lib/scribble/racket.rkt

```diff
@@ -326,6 +326,7 @@
            [first (if escapes?
                       (syntax-case c (code:line)
                         [(code:line e . rest) #'e]
+                      [(code:line . rest) #'rest]
```

Comment:
I _think_ this case only matches `(code:line)`, so it shouldn't hurt but also shouldn't change how anything looks.